tests/Integration/ForceFactoriesTraitUsage/kernel_test_case_with_only_reset_database_trait.phpt

```diff
@@ -0,0 +1,38 @@
+--TEST--
+phpunit --version
+--FILE--
+<?php
+$_ENV['KERNEL_CLASS'] = 'Zenstruck\Foundry\Tests\Fixture\TestKernel';
+$_SERVER['argv'][] = '--do-not-cache-result';
+$_SERVER['argv'][] = '--no-configuration';
+$_SERVER['argv'][] = __DIR__ . '/../../Fixture/ForceFactoriesTraitUsage/KernelTestCaseWithOnlyResetDatabaseTraitTest.php';
```

Comment:
for this specific case, where only the `ResetDatabase` trait is used, the error occurs during the "before test class" hook, so the error cannot be caught in a regular test . Hence this low level ".phpt" test